### ISSUE TITLE
little help installing

### ISSUE THREAD

**Issue by NightBarone** [Coombaa/TTAutoRecord#44]
hi and thanks for the work. I just have a couple of questions.

what is the purpose of firefox?
when we sign to tiktok, do we sign in in chrome or firefox? (because edit cockies extension in chrome)
where to paste this line to install modules "  pip install -r requirements.txt " ?

Thanks

**Comment by shadowdragon245** [comment]
bit late but do you still need help?

**Comment by NightBarone** [comment]
yes please. is there any video or step by step guide that would be great .
Thanks mate.
